### ISSUE TITLE
NoInfoAvailableException when attempting to investigate an unavailable/restricted user

### ISSUE THREAD

**Issue by myangelaku** [circleguard/circleguard#113]
```
[ERROR] 2020/12/13 17:24:38.0580 Error while running circlecore. Please report this to the developers through discord or github.
 (circleguard_gui, main_tab.py:516)
Traceback (most recent call last):
  File "gui\main_tab.py", line 308, in run_circleguard
  File "site-packages\circleguard\circleguard.py", line 465, in load_info
  File "site-packages\circleguard\loadables.py", line 138, in load_info
  File "site-packages\circleguard\loadables.py", line 327, in load_info
  File "site-packages\circleguard\loader.py", line 124, in wrapper
  File "site-packages\circleguard\loader.py", line 349, in get_user_best
  File "site-packages\circleguard\loader.py", line 610, in check_response
circleguard.loader.NoInfoAvailableException: No info was available from the api for the given arguments.
```
Attempted to investigate user 5645067 if it matters

**Comment by tybug** [comment]
happens in the `cg.load_info` call for any replay container (map, user, mapuser)

**Comment by tybug** [comment]
closed by 822a5f625bc5eaa7d3a3ca5c3b3e025c47422509
